src/lib.rs

```diff
@@ -969,3 +1229,22 @@ pub fn clang_version() -> ClangVersion {
         full: raw_v.clone(),
     }
 }
+
+/// Test command_line_flag function.
+#[test]
+fn commandline_flag_unit_test_function() {
+
+    let bindings = ::builder();
+    let command_line_flags = bindings.command_line_flags();
+
+    let test_cases = vec!["--no-derive-default",
+                          "--generate",
+                          "function,types,vars,methods,constructors,destructors"]
+                    .iter()
+                    .map(|&x| x.into())
+                    .collect::<Vec<String>>();
+
+    assert!(test_cases.iter().all(|ref x| command_line_flags.contains(x)) );
```

Comment:
Please add another test that exercises a non-default builder. Something with some whitelisting and opaque-ing perhaps?